client/src/Components/Layouts/Order/addOrder.jsx

```diff
@@ -0,0 +1,418 @@
+import antd, { Icon } from "antd";
+import React from "react";
+import validator from "validator";
+import "./style.css";
+const { Button, Modal, Form, Input, Select, AutoComplete, notification } = antd;
+const { Option } = Select;
+let id = 0;
+const openNotificationWithIcon = (type, message) => {
+  notification[type]({
+    message
+  });
+};
+const CollectionCreateForm = Form.create({ name: "form_in_modal" })(
+  // eslint-disable-next-line
+  class extends React.Component {
+    state = {
+      dataSourceCaptains: [],
+      dataSourcePlaces: [],
+      error: {
+        errCaptain: "",
+        errPalce: ""
+      }
+    };
+    componentDidMount() {
+      fetch("/api/v1//getCaptainsNames")
+        .then(res => res.json())
+        .then(res => {
+          const { error } = res;
+          if (error) {
+            this.openNotificationWithIcon("error", error);
+          } else {
+            this.setState({ dataSourceCaptains: res.result });
+          }
+          return fetch("/api/v1/getPlacesNames");
```

Comment:
@denaHS You could actually do both fetches in parallel, instead of waiting for getCaptainsNames to finish before starting getPlacesNames.  That would make this component load faster for the user.  I'd suggest putting each fetch and then-handler into its own function, and then calling them both from componentDidMount():
```
componentDidMount() {
  this.loadCaptainsNames()
  this.loadPlacesNames()
}
```